src/tito/tagger/main.py

```diff
@@ -405,7 +405,7 @@ def _bump_version(self, release=False, zstream=False, force=False):
                                         increase_zstream(match.group(2)),
                                         "\n"
                         ))
-                elif force:
+                elif self._use_version is not None:
```

Comment:
Am I correct that a user who has a custom tagger defined today (a custom tagger for example), if they are using --use-version their tagger implementation gets changed on the fly and uses ForceVersionTagger, resulting in the specific version they set being tagged.

With this change, it looks like if they kept doing this it will now just call their normal configured tagger by default, which probably ignores the version specified but silently works and does it's normal thing.